nextcord/guild.py

```diff
@@ -2580,10 +2589,10 @@ async def create_role(
         mentionable: :class:`bool`
             Indicates if the role should be mentionable by others.
             Defaults to ``False``.
+        icon: Optional[Union[:class:`str`, :class:`bytes`, :class:`Asset`, :class:`Attachment`]]
```

Comment:
If it is left alone, then `File` should be added here. If `File` is removed, as proposed above, then this is fine.